README.md

```diff
@@ -75,13 +75,6 @@ For more information on using `tsconfig.json` refer to the [TypeScript documenta
 }
 ```
 
-## Issues
-
-Report issues with next version at https://github.com/Esri/feedback-js-api-next/issues/new/choose
-
-* Bugs: Bug reports should be specific to js.arcgis.com/next and not present in the current API release.
-* Enhancements: For this initiative, enhancement requests should be specific to a feature in js.arcgis.com/next.
-
```

Comment:
Why are we removing this whole section? I think we still want people to report bugs... but we can remove the bullet point on enhancements?

Reply:
They can still do that, the reporting option isn't going away. If they go to the Issue tab => New Issues they'll see those various choices. This PR only removes the Enhancement template. 

Going to the Issues tab is a well-known github pattern for reporting problems...or maybe I misunderstood your point?

Reply:
yep I guess I didn't understand why you were removing this entire section from the README instead of just the bullet point about enhancements.

Reply:
makes sense - thanks!

Reply:
Thanks for the feedback, Anne!